com.vrcfury.vrcfury/Editor/VF/Builder/Exceptions/VRCFExceptionUtils.cs

```diff
@@ -18,19 +18,21 @@ public static class VRCFExceptionUtils {
             try {
                 await go();
             } catch(Exception e) {
+                e = GetGoodCause(e);
```

Comment:
I don't think this is proper. GetGoodCause eliminates part of the stack trace involved in the invocation of the method. The only reason we used it below is because invocation exception shave a dirty version of the original exception message.

Reply:
hmm, i didn't see much value in keeping the invokes, though i suppose how during certain actions in development it could be useful.

Reply:
Did you mean to close this? I think the rest is reasonable 😅

Reply:
the rest of the changes are super minor, but sure!